Makefile

```diff
@@ -0,0 +1,5 @@
+load-smoke-test:
+	locust -f tests/load/smoke.py --host=https://developer.allizom.org
+
+browser-tests:
+	cd tests/ui ; ./node_modules/.bin/intern-runner config=intern-local d=developer.allizom.org b=firefox
```

Comment:
Ah, one thing, could you cd back to the main directory? Or use [pushd/popd](https://en.wikipedia.org/wiki/Pushd_and_popd)?